test/extended/templates/templateservicebroker_e2e.go

```diff
@@ -72,6 +75,10 @@ var _ = g.Describe("[templates] templateservicebroker end-to-end test", func() {
 		})
 		o.Expect(err).NotTo(o.HaveOccurred())
 
+		// give test user access to the openshift project's templates ... now needed with provision's SAR for get on templates (previously commented out)
+		err = cli.AsAdmin().Run("policy").Args("add-role-to-user", "edit", cli.Username(), "-n", "openshift").Execute()
```

Comment:
its been interesting ... I needed to explicitly add view (I had changed this to just view but must not have committed it) when running the extended test locally.  If I did not, I got the permission error the comment alluded to.

And when running similar tests from from the command line, I had to given developer view.

But when I run the clusterup.sh extended tests, I did not need this.